saleor/plugins/webhook/tests/subscription_webhooks/payloads.py

```diff
@@ -34,6 +34,18 @@ def generate_attribute_payload(attribute):
     )
 
 
+def generate_attribute_value_payload(attribute_value):
+    return json.dumps(
+        {
+            "attributeValue": {
+                "name": attribute_value.name,
+                "slug": attribute_value.slug,
+                "value": attribute_value.value,
```

Comment:
What about other fields that hold values for other data types, e.g. rich/plain text dates etc? Maybe we don't need it because IIRC for some of the types we store their value in `slug` anyway, but I'm not 100% sure.

Reply:
its just some random fields that I'm using only in tests, if you want I can add more

Reply:
👍